target/linux/brcmiproc/bcm58800/profiles/00-default.mk

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (C) 2017 Lede
```

Comment:
You are not Lede, so either put your name here or remove the Copyright entirely for this trivial file.

Reply:
Added my name.